api-reference/endpoint/get_item.mdx

```diff
@@ -1,85 +1,59 @@
 ---
 
-### Body
-
-<ParamField body="name" type="string">
-  This is the name of the user group.
-</ParamField>
-
-<ParamField body="user_group_id" type="string">
-  This is the ID you use to identify this item group in your database.
-</ParamField>
-
-<ParamField body="mapping" type="object">
-  This is a JSON mapping of schema id to either the data source that this user
-  group should be associated with or id of the datasource you provided when
-  creating it.
-</ParamField>
-
-<ParamField body="properties" type="object">
-  This is a JSON object for properties assigned to this user group. These will
-  be accessible through variables in the dashboards and SQL editor
+### Parameters
+<ParamField path="itemId" type="string" required>
+  This is the Id of the item.
 </ParamField>
 
 ### Response
 
-<ResponseField name="success" type="number">
-  Indicates whether the call was successful. 1 if successful, 0 if not.
+<ResponseField name="itemName" type="strig">
+  Name of Item
 </ResponseField>
 
-<ResponseField name="new_user_group" type="boolean">
-  Indicates whether a new user group was created.
+<ResponseField name="ownerId" type="string">
+  Owner Id of Item
 </ResponseField>
 
-<ResponseField name="user_group" type="object">
-
-The contents of the user group
-
+<ResponseField name="characteristics" type="object">
+  Characterstics of Item with Blockchain Id
 <Expandable title="Toggle object">
 
-<ResponseField name="team_id" type="number">
-  This is the internal ID for this user group. You don't need to record this
-  information, since you will not need to use it.
-</ResponseField>
-
-<ResponseField name="token" type="string">
-  This is the user group token (userGroupToken or USER_GROUP_TOKEN) that will be
-  used to identify which user group is viewing the dashboard. You should save
-  this on your end to use when rendering an embedded dashboard.
+<ResponseField name="characteristicOne" type="string">
+  Item Characterstic One
 </ResponseField>
 
-<ResponseField name="name" type="string">
-  This is the name of the user group provided in the request body.
+<ResponseField name="characteristicTwo" type="string">
+  Item Characterstic Two
 </ResponseField>
 
-<ResponseField name="provided_id" type="string">
-  This is the user_group_id provided in the request body.
+<ResponseField name="characteristicThree" type="string">
+  Item Characterstic Three
 </ResponseField>
 
-<ResponseField name="properties" type="JSON or null">
-  This is the properties object if it was provided in the request body
+<ResponseField name="blockchainId" type="string">
+  Id from Blockchain
 </ResponseField>
 
 </Expandable>
 
+<ResponseField name="itemId" type="string">
+  Id of the Item
+</ResponseField>
+
 </ResponseField>
 
 <RequestExample>
 
 ```bash Example Request
-curl --location --request GET 'https://api.mintlify.com/api/user' \
+curl --location --request GET \
+'https://api.mintlify.com/api/v1/items/:itemId' \
```

Comment:
we should be calling our api url @jaydeep0621

Reply:
Ok let me update it

Reply:
![Uploading image.png…]()
Updated API URL